deploy/templates/cert-manager/crd_webhooks_patch.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           path: /convert
           port: 443
         # caBundle will be filled by cert-manager on creation
-        caBundle: Cg==
+        # caBundle: Cg==
```

Comment:
Maybe we should just remove this line (as well as the other mentions of `caBundle: Cg==` from the templates) rather than commenting it out?

Reply:
I removed them in my latest commit